### PR TITLE
Fix: Gracefully handle malformed CSS

### LINKED ISSUE #1580: [parser-css] Use postcss-safe-parser for fault tolerance
Currently malformed CSS will cause the entire file to be ignored. By switching to use the fault-tolerant parser for PostCSS such files can still be analyzed, behaving more like a browser would.

https://github.com/postcss/postcss-safe-parser

### PR DESCRIPTION
**Pull request by antross**
## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- ~Added/Updated related documentation.~
- [x] Added/Updated related tests.

## Short description of the change(s)

Fix #1580

Also update tests to run in parallel.

Co-Authored-By: Borja Ruiz Torres <borgitas21@users.noreply.github.com>

### CODE REVIEW

**Comment by antross** [comment]
@sarvaje This will conflict with your PR #1768 since I already refactored the `parser-css` tests to remove `.serial` in this PR. Assuming you're ok with the changes I made here, I recommend merging this PR first, then updating #1768 to use the changes from this PR for `parser-css` tests instead.

File: packages/parser-css/src/parser.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const safe = require('postcss-safe-parser');
```
Comment (sarvaje) [inline packages/parser-css/src/parser.ts]: Any reason to use require instead of import?

**Comment by molant** [comment]
Let's merge this one then. I need #1768 merged as well to see if #1758 improves in Windows as well before doing more changes there

File: packages/parser-css/src/parser.ts (hunk 1 above)
Comment (antross) [reply packages/parser-css/src/parser.ts]: The usual: `postcss-safe-parser` returns a function.

Comment (sarvaje) [reply packages/parser-css/src/parser.ts]: ok!

**Pull request merged**